app/editor/EditorCtrl.js

```diff
@@ -190,6 +220,45 @@
       EventService.emitEvent(EventService.EVENT.GOTO_PREV_PAGE);
     };
 
+    editorCtrl.resetFilter = function() {
+      resetFilter(true);
+    };
+
+    editorCtrl.updateFilter = function() {
+      updateFilter(true);
+    };
```

Comment:
You could also encode the default value in the function definitions, so that here you could have `editorCrtl.updateFilter = updateFilter;`. You would just need to add a line at the start of the other function, like this:

```
function updateFilter(fireEvent) {
  fireEvent = arguments.length === 0 ? true : fireEvent;
```